src/core/server/app/middleware/commentEmbedWhitelisted.ts

```diff
@@ -37,6 +37,14 @@ export const commentEmbedWhitelisted =
             origin = req.header("Origin");
           }
           if (origin) {
+            // if oEmbed API call, we also check oEmbed allowed origins on tenant
+            if (oembedAPI) {
+              if (
+                tenant.embeddedComments?.oEmbedAllowedOrigins.includes(origin)
+              ) {
+                return next();
+              }
+            }
```

Comment:
since both of these nested `if`'s have no `else` how do you feel about doing the following instead?

```
// if oEmbed API call, we also check oEmbed allowed origins on tenant
      if (oembedAPI && tenant.embeddedComments?.oEmbedAllowedOrigins.includes(origin)) {
        return next();
      }
 ```
 
I only suggest this because I'm looking at the full file and seeing that this we're at 6 levels of if-nesting happening. 

Not sure if we want to continue to add to it. It might be time for a refactor before this middleware gets out of hand.

Thoughts?

Reply:
yep, updated--thank you for calling out!